apps/meteor/app/autotranslate/server/deeplTranslate.ts

```diff
@@ -13,6 +12,7 @@ import type {
 	ITranslationResult,
 	ISupportedLanguage,
 } from '@rocket.chat/core-typings';
+import { serverFetch as fetch } from '@rocket.chat/server-fetch';
```

Comment:
is not the ideia of exporting `serverFetch` avoid use the same `fetch` name? why not keep `serverFetch` everywhere?

Reply:
Think it's more a preference thing. Part of my goal with the exported name is that ppl using it remember is not the standard fetch.

Renaming it to fetch also makes "easier" to move away from it when we use node's built in fetch in the future.